go/oasis-test-runner/cmd/root.go

```diff
@@ -47,13 +57,30 @@ var (
 		Run:   runList,
 	}
 
-	rootFlags = flag.NewFlagSet("", flag.ContinueOnError)
+	rootFlags       = flag.NewFlagSet("", flag.ContinueOnError)
+	TestParamsFlags = flag.NewFlagSet("", flag.ContinueOnError)
```

Comment:
Why are these public?

Reply:
I use the same `--params.X.Y` syntax in `ba` tool as well to fetch the benchmark instance with specific parameter set.

Reply:
This was fixed by moving `cmp` command to `oasis-test-runner`. FlagSets are now back private.